capstone-frontend/src/components/ListPage.jsx

```diff
@@ -110,16 +111,29 @@ class ListPage extends Component {
           <div>
             <Row>
               <Col>
-                <h2 className="mt-4 ml-2">{this.state.bookList.name}</h2>
-              </Col>
-              <Col className="text-right">
-                <SearchBookModal
-                  objectId={this.props.match.params.id}
-                  update={this.updateListPage}
-                  putURL="/api/booklist"
-                  type="booklist"
-                  btnStyle="btn btn-primary mt-4 mr-2" />
+                <h2 className="ml-2">{this.state.bookList.name}</h2>
+                {this.state.bookList.userID === window.localStorage.getItem('userID') ?
+                  <h5 className="mb-1 ml-2 text-muted">Owner</h5>
+                  : 
+                  <h5 className="mb-1 ml-2 text-muted">Collaborator</h5>}
               </Col>
+              {this.state.bookList.userID === window.localStorage.getItem('userID') &&
+                <Col className="margin-auto p-0 mr-3">
+                  <div id="modal-buttons" className="mx-2">
+                    <SearchUserModal
+                      type='booklists'
+                      bookList={this.state.bookList}
+                      text='Search/View Collaborators'
+                      checkoutText='Current Collaborators'
+                      btnStyle="btn btn-primary mx-3 h-100" />
+                    <SearchBookModal
+                      objectId={this.props.match.params.id}
+                      update={this.updateListPage}
+                      putURL="/api/booklist"
+                      type="booklist"
+                      btnStyle="btn btn-primary h-100" />
+                  </div>
+                </Col>}
             </Row>
             <hr className="light-gray-border mx-2 my-2" />
             <h3 className="text-center mt-4">Booklist has No Books</h3>
```

Comment:
nit: nit: Use single quotations for string literals, a mix of single quotes and double quotes here

Reply:
Ok